CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+- `QasSelect`: Added fetch options when component is created on screen. 
+
+### Added
+- `QasSelect`: Added pagination validation with `hasNextPage` for cases where there is no items count.  
```

Comment:
- `QasSelect`: Added pagination validation with `hasNextPage` for cases where there is no count on response of action `fetchFieldOptions`.